<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordSafeDialog</class>
 <widget class="QDialog" name="PasswordSafeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password safe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/CS_icon.png</normaloff>:/images/CS_icon.png</iconset>
  </property>
  <widget class="QPushButton" name="ButtonSendpassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSendLoginname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send loginname</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSendPW_LN">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send PW tab LN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOk">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDelay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>500</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>500</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioCutUPaste">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> Cut &amp;&amp; paste</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioKeyboard">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> via Keyboard</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
